config/default.cfg

```diff
@@ -25,10 +25,10 @@ cfg$description <- "REMIND run with default settings"
 cfg$regionmapping <- "config/regionmappingH12.csv"
 
 #### Current input data revision (<mainrevision>.<subrevision>) ####
-cfg$inputRevision <- 6.316
+cfg$inputRevision <- "6.320"
```

Comment:
why is there also a new input data revision?

Reply:
Because I got confused in all the back and forth with Robin …